### DISCUSSION TITLE
How to select differential regulons between to conditions in a cell type specific matter

### DISCUSSION THREAD

**Discussion by jeji0209** [aertslab/SCENIC#372] [* General SCENIC questions]
Hi, I have a dataset with multiple cell types, and all cells are divided by 2 conditions.
What I want to do is find a condition specific regulons within a certain cell type. 
For example, I want to find condition1 specific regulons in cell type B.
I am thinking of three methods currently.

1. compare the AUC activity score of the two cell clusters of my interest using statistic methods such as Wilcoxon test. 
(condition 1, cell type B cells vs. condition 2, cell type B cells)

2. make new cell clusters, considering both condition and cell type. (e.g. cluster1 - cell type A + condition1, cluster2 - cell type A + condition 2, cluster3- cell type B + condition1, cluster4 - cell type B + condition2 , and so on...). Then calculate RSSscores for my newly made clusters.

3. Splitting dataset by condition and running SCENIC seperately, then comparing the results.

Which of the three methods is most appropriate? of any other new ways to suggest?
Thanks in advance :)

**Comment by HDaaboul** [comment]
Hello! 
I have the same issue now, could you tell me which of these approaches worked best for you? 
Thank you in advance!
@jeji0209

**Comment by The-G-Cat** [comment]
I'm not sure if I have correctly understood your idea. It seems that you have several sets of scRNA-seq data from cells treated with different methods, and now you want to find out the differences in regulons among different treatments for a certain cell population in one of the treatment groups?

If my understanding is correct, my approach would be as follows: You can integrate your data, label the experimental conditions for the cells, and then integrate them into a dataframe object (or a Seurat object, depending on whether you are using R or Python). After clustering, either analyze the entire group or the specific groups you need using PyScenic. Then, during the statistical analysis, integrate the experimental condition labels within each cluster for classification and plotting. This should allow you to observe the TF situation of the same type of cells under different experimental conditions (remember to calculate the Z-score).

I'm doing something similar. What kind of cells did you use? Is it a mouse? I hope that after you obtain the results, you can also answer the following questions for me! Thank you very much!
